restws_basic_auth/restws_basic_auth.install

```diff
@@ -6,14 +6,17 @@
  */
 
 /**
- * Migrate variables to config.
+ * Implements hook_uninstall().
  */
-function restws_basic_auth_update_1000() {
-  // Migrate variables to config.
-  $config = config('restws.settings');
-  $config->set('restws_basic_auth_user_regex', update_variable_get('restws_basic_auth_user_regex', '/^restws.*/'));
-  $config->save();
+function restws_basic_auth_uninstall() {
+  // Remove the username regex.
+  config_set('restws.settings', 'restws_basic_auth_user_regex', NULL);
```

Comment:
Is this needed? Since you declared restws_config_info, the config files will be removed automatically upon installation, so settings this to null doesn't make sense.